frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -25,6 +25,7 @@ export class StorageClassDropdownInner extends React.Component<
 
   componentWillReceiveProps(nextProps) {
     const { loaded, loadError, resources } = nextProps;
+    const { filter = (args) => (args ? true : false) } = this.props;
```

Comment:
You don't really need a default filter. Simply check below `if (!filterr || filter(...`